.circleci/config.yml

```diff
@@ -286,6 +283,7 @@ jobs:
       - run:
           name: Run tests
           command: |
+            touch env.d/development/crowdin && \
```

Comment:
Just curious why do we need this empty files for tests? Should we add a comment to justify?

Reply:
this file is used by crowdin in the `docker-compose` config file. Without it `docker-compose` will stop with an error saying that the file does not exists.

Reply:
@lunika I get he same error locally when running make bootstrap. Shouldn't you add a cp crowdin.dist in the make file?